### PR TITLE
[BACK-51] Add Contributor Count

### PR DESCRIPTION
**Pull request by 2015builder**
### Description
Added the new ContributorCount

### How I implemented
New method how Clemens explained it. Get the names of all people that made a commit to the main branch and eliminate duplicates

### Other

### CODE REVIEW

File: packages/graphql_server/src/types.ts
Comment (thomas-woehrle) [inline packages/graphql_server/src/types.ts:1]: I have already refactored all of these types into the respective files of the `/types` folder and updated the imports that existed before your PR.
So delete this file and update the imports where/if needed, thanks

File: packages/graphql_server/src/scraping/githubScraping.ts
Comment (thomas-woehrle) [inline packages/graphql_server/src/scraping/githubScraping.ts:1]: I would move it into the api/githubApi.ts file.
I think there is no scraping going on but rather an API call

File: packages/graphql_server/types/githubScraping.d.ts
Comment (thomas-woehrle) [inline packages/graphql_server/types/githubScraping.d.ts:1]: If you move the other file to `/api/githubApi.ts`, please move these types into the corresponding types file as well, thanks

**Review by thomas-woehrle — CHANGES_REQUESTED** [review]
Great job, just small things.
Your build fail message indicates that you did not run prettier. You can so with` npm run format`.

File: packages/graphql_server/src/scraping/githubScraping.ts [hunk 1]
```diff
@@ -100,3 +101,69 @@ export async function fetchTrendingDevelopers(timeMode: timeMode) {
...
+    repo
+  }
+
+  try {
+    const response: AxiosResponse<ContributorResponse> = await axios.post(
+      'https://api.github.com/graphql',
+      { query, variables },
+      {
+        headers: {
+          Authorization: `Bearer ` + authToken
```
Comment (cnk1dev) [inline packages/graphql_server/src/scraping/githubScraping.ts:154]:
```suggestion
          Authorization: `Bearer ${authToken}`
```

**Review by cnk1dev — CHANGES_REQUESTED** [review]
Refactor according to @thomas-woehrle's comments & run prettier to pass GitHub tests, rest looks good.

**Comment by thomas-woehrle** [comment]
I merged main

**Review by thomas-woehrle — APPROVED** [review]

File: packages/graphql_server/src/api/githubApi.ts [hunk 2]
```diff
@@ -65,3 +71,69 @@ export async function getUserInfo(query: string, authToken: string): Promise<Git
...
+      }
+    }
+  `
+
+  const variables = {
+    owner,
+    repo
+  }
+
+  try {
```
Comment (pczern) [inline packages/graphql_server/src/api/githubApi.ts:118]: don't catch errors, so we can catch them in the GraphQL resolvers and give appropriate feedback and logging where due

**Review by pczern — CHANGES_REQUESTED** [review]

**Review by pczern — APPROVED** [review]

**Review by cnk1dev — APPROVED** [review]

**Pull request merged**